tools/acrn-manager/Makefile

```diff
@@ -1,11 +1,20 @@
 
 OUT_DIR ?= .
 
-all: acrnctl.c
+all: $(OUT_DIR)/libacrn-mngr.a $(OUT_DIR)/acrnctl
+
+$(OUT_DIR)/libacrn-mngr.a: acrn_mngr.c acrn_mngr.h
+	$(CC) -c acrn_mngr.c -DMNGR_DEBUG -I../../devicemodel/include -Wall -g
```

Comment:
Could we make this macro (`-DMNGR_DEBUG`) conditional based on the value of `RELEASE`?

Not in this PR obviously since it's been merged but as an enhancement?